src/lane_manager.cpp

```diff
@@ -270,6 +271,9 @@ void LaneManager::add_car()
 {
 	std::map<direction, Lane*>::iterator it = m_lanes.begin();
 	if (game_level == 1) {
+		std::advance(it, rand() > RAND_MAX/2? 1:3);
+	}
+	else if (game_level == 2) {
```

Comment:
Found a small issue with choosing the lanes randomly, which is that the lanes in play will sometimes change in between level 1 and 2 (so for example, level 1 I had east and west active, then level 2 I had north, east, and south active. The west lane got switched "off" during the level change, but I still had some remaining cars in that lane, so I had four lanes with cars in level 2 until I cleared the west lane.) I think it would probably be better to just hardcode in specific lanes to use in level 1 and 2.